pkg/proc/dwarf_export_test.go

```diff
@@ -30,3 +31,14 @@ func NewCompositeMemory(p *Target, pieces []op.Piece, base uint64) (*compositeMe
 func IsJNZ(inst archInst) bool {
 	return inst.(*x86Inst).Op == x86asm.JNE
 }
+
+// HasDebugPinner returns true if the target has runtime.debugPinner.
+func (bi *BinaryInfo) HasDebugPinner() bool {
```

Comment:
Maybe this should be promoted outside of test cases since it's used elsewhere.

Reply:
Where?

Reply:
https://github.com/go-delve/delve/pull/3787/files#diff-59fa7be9add4e1bb71e059c33240b9d77b645837db52c32a18810508d47631e9R320

https://github.com/go-delve/delve/pull/3787/files#diff-b6813f76b09053fe4467f8fe8f18410ca57cca127343a973a43eb9a9de2ce590R187

Reply:
I've added hasDebugPinner to BinaryInfo. IMO this isn't worth exporting publicly.

Reply:
SGTM